### DISCUSSION TITLE
Possible to change settings?

### DISCUSSION THREAD

**Discussion by joshuaholmes90** [nuxt-community/firebase-module#498] [Q&A] [answered]
Hey All,

Using this module and it's great, is it possible to customize the settings of firestore even further?

```
firebase.firestore().settings({
  experimentalForceLongPolling: true
});
```

^^ For example I would like to do this

**Comment by mati0090** [comment, marked as answer]
Hey, just use `settings` key for your `firestore` configuration like so:
```
firestore: {
  emulatorPort: 8080,
  emulatorHost: 'localhost',
  settings: {
    experimentalForceLongPolling: true
  }
}
```

**Reply by joshuaholmes90** [reply]
Great Reply - Thank you for your help!
